homework-solutions/js/hw-3/task-2.js

```diff
@@ -4,3 +4,8 @@
 n + nn + nnn, где n не перемножаются, а конкатенируются
 
 */
+
+let n = 2;
+let result = +String(n) + +((String(n) + String(n))) + +(String(n) + String(n) + String(n));
```

Comment:
- Достаточно один параметр привести в строку для получения конкатенации
- Полученное значение в переменной `result` (так же и `n`) не переопределяется - используем `const`

```suggestion
const result = +String(n) + +(String(n) + n) + +(String(n) + n + n);
```